kroxylicious-operator/src/test/resources/DerivedResourcesTest/clusterref-resource-not-found/cond-ResolvedRefs-bar.yaml

```diff
@@ -6,7 +6,7 @@
 
 ---
 cluster: "bar"
-type: "Accepted"
+type: "ResolvedRefs"
```

Comment:
Should the cluster be only `ResolvedRefs: False`, or should it also have an `Accepted: False` condition? The resolution failure also implies we won't accept the cluster.

Reply:
I think it is up to us to define what conditions appear on which resources and what subsequently happens or doesn't happen (though in practice we will be influenced by Gateway API). Looking at https://github.com/kubernetes-sigs/gateway-api/blob/main/site-src/guides/implementers.md?plain=1#L147C4-L147C16 I think in this case `ResolvedRefs: false` means `Accepted: true` is impossible, because it's impossible to know where to "write the config". 

The question then is whether we should be explicit by adding `Accepted: false` as well, or whether `ResolvedRefs: false` is/ought to be enough to convey that this baby ain't gonna fly. I don't have a objective answer. But having a single condition is feels simpler to understand and manage.